vllm/platforms/cuda.py

```diff
@@ -74,6 +74,7 @@ class CudaPlatformBase(Platform):
     _enum = PlatformEnum.CUDA
     device_type: str = "cuda"
     dispatch_key: str = "CUDA"
+    is_async_output_support: bool = True
```

Comment:
I think this is a bit misleading, maybe we should not use inheritance for this.

Reply:
Just curious, would this be different in nvml and non-nvml?

Reply:
> I think this is a bit misleading since it's not always `True`, maybe we should not use inheritance for this.

Just curious, would this be different in nvml and non-nvml?

Reply:
sorry, I'm not very familiar with CUDA, could you tell me what kind of case that the `async_output` is not True? I just add the `enforce_eager` special case  below in `is_async_output_supported` func

Reply:
I mean that the variable name is a bit misleading because it's overridden to `False` in the case that `enforce_eager` is set.